Sources/Run_in_Sandbox/RunInSandbox.ps1

```diff
@@ -383,29 +383,33 @@ switch ($Type) {
 
         $add_parameters.add_click({
                 $Script:Paramaters = $parameters_to_add.Text.ToString()
-                $Script:Startup_Command = "wscript.exe $Full_Startup_Path $Paramaters"
+                $Script:Startup_Command = "wscript.exe $Full_Startup_Path_Quoted $Paramaters"
                 $Form_VBS.close()
             })
 
         $Form_VBS.Add_Closing({
                 $Script:Paramaters = $parameters_to_add.Text.ToString()
-                $Script:Startup_Command = "wscript.exe $Full_Startup_Path $Paramaters"
+                $Script:Startup_Command = "wscript.exe $Full_Startup_Path_Quoted $Paramaters"
             })
 
         $Form_VBS.ShowDialog() | Out-Null
 
         New-WSB -Command_to_Run $Startup_Command
     }
     "ZIP" {
-        $Script:Startup_Command = $PSRun_Command + " " + "Expand-Archive $Full_Startup_Path $Sandbox_Desktop_Path\ZIP_extracted"
+        $command = "Expand-Archive -LiteralPath '$Full_Startup_Path' -DestinationPath '$Sandbox_Desktop_Path\ZIP_extracted'"
+        $bytes = [System.Text.Encoding]::Unicode.GetBytes($command)
+        $encodedCommand = [Convert]::ToBase64String($bytes)
+
+        $Script:Startup_Command = "C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -sta -WindowStyle Hidden -NoProfile -ExecutionPolicy Unrestricted -EncodedCommand " + $encodedCommand
         New-WSB -Command_to_Run $Startup_Command
     }
 }
 
 Start-Process $Sandbox_File_Path -Wait
 
 if ($WSB_Cleanup -eq $True) {
-    Remove-Item -Path $Sandbox_File_Path -Force -ErrorAction SilentlyContinue
+    Remove-Item -LiteralPath $Sandbox_File_Path -Force -ErrorAction SilentlyContinue
```

Comment:
I see why you used LiteralPath here, but why not for the other "Remove-Item" command below? Wouldnt it make sense to add it there aswell?

Reply:
Yes, it likely would make sense to change it there as well - the only reason I didn't change those lines was because I was hesitant to make changes related to functions that I don't use or test (e.g. anything related to Intune). But if you don't see any issues with it, then feel free to add. 😊

Reply:
I´ll just leave it as it is now, i test it and see if it makes any difference